tooling/nargo_fmt/tests/expected/block.nr

```diff
@@ -0,0 +1,20 @@
+fn main(mut x: u32, y: u32, z: u32) {
+    x = unsafe { x + y };
+    assert(
+        unsafe {
+        x == z
+    }
+    );
```

Comment:
not ideal formatting but can be addressed later.

Reply:
@noir-lang/core I checked and in general a block statement inside a function argument doesn't get formatted right:

```rust
fn main() {
    let x = 1;
    let z = 1;
    assert(
        {
        x == z
    }
    );
}
```

and the same happens with `comptime` and `unsafe`. Should I still try to fix that in this PR?

Reply:
If it's not an easy fix we shouldn't bother in this PR. It's an existing issue with comptime as well then so that's enough reason to leave it for a separate PR.

Reply:
Sounds good, let's leave it for a separate PR. I tried to look at the formatter but I don't understand it well yet.